### PR TITLE
Bring top-level `steps` and `groupStep` `steps` into parity

### PR DESCRIPTION
**Pull request by thejcannon**
- Adding some missing step types to `groupStep`
  -  I didn't validate these were valid. LMK if you want me to or you plan to
- Re-ordered them to be consistent

~LMK if you think it's worth one or more tests~

### CODE REVIEW

**Comment by thejcannon** [comment]
Actually I wouldn't feel good about this without adding a unit test and demo-ing an e2e test. So let's wait for that.

File: schema.json [hunk 1]
```diff
@@ -1322,18 +1322,19 @@
...
           "items": {
             "anyOf": [
-              { "$ref": "#/definitions/stringBlockStep" },
               { "$ref": "#/definitions/blockStep" },
-              { "$ref": "#/definitions/commandStep" },
-              { "$ref": "#/definitions/nestedCommandStep" },
-              { "$ref": "#/definitions/triggerStep" },
-              { "$ref": "#/definitions/nestedTriggerStep" },
-              { "$ref": "#/definitions/stringInputStep" },
+              { "$ref": "#/definitions/nestedBlockStep" },
```
Comment (thejcannon) [inline schema.json:1326]: This is new

Comment (thejcannon) [reply schema.json:1326]: Yeah OK tried this and it works 😄

File: schema.json [hunk 2]
```diff
@@ -1322,18 +1322,19 @@
...
+              { "$ref": "#/definitions/stringBlockStep" },
               { "$ref": "#/definitions/inputStep" },
               { "$ref": "#/definitions/nestedInputStep" },
-              { "$ref": "#/definitions/stringWaitStep" },
+              { "$ref": "#/definitions/stringInputStep" },
+              { "$ref": "#/definitions/commandStep" },
+              { "$ref": "#/definitions/nestedCommandStep" },
               { "$ref": "#/definitions/waitStep" },
               { "$ref": "#/definitions/nestedWaitStep" }
```
Comment (thejcannon) [inline schema.json:1334]:
```suggestion
              { "$ref": "#/definitions/nestedWaitStep" },
```

File: schema.json [hunk 3]
```diff
@@ -1322,18 +1322,19 @@
...
+              { "$ref": "#/definitions/stringWaitStep" },
+              { "$ref": "#/definitions/triggerStep" },
+              { "$ref": "#/definitions/nestedTriggerStep" },
```
Comment (thejcannon) [inline schema.json:1337]:
```suggestion
              { "$ref": "#/definitions/nestedTriggerStep" }
```

**Review by DrJosh9000 — APPROVED** [review]
LGTM

**Pull request merged**